frontend/apps/reader/modules/readergesture.lua

```diff
@@ -1518,6 +1526,11 @@ function ReaderGesture:pageUpdate(page)
 
 end
 
+function ReaderGesture:onIgnoreHoldCorners(ignore_hold_corners)
+    self.ignore_hold_corners = ignore_hold_corners
```

Comment:
Might be that event/handler  could be named `onIgnoreHoldCornersToggled` and it would `G_reader_settings:isTrue("ignore_hold_corners")` itself, so sender/caller do not have to provide it as an argument to the event?

Reply:
Makes sense, but I'd still prefer an optional argument to explicitly set it on or off.